eruditus/lib/discord_util.py

```diff
@@ -0,0 +1,246 @@
+import re
+from datetime import datetime
+from typing import Optional
+
+import aiohttp
+import discord
+
+from config import (
+    CHALLENGE_COLLECTION,
+    CTF_COLLECTION,
+    DBNAME,
+    MAX_CONTENT_SIZE,
+    MONGO,
+    TEAM_NAME,
+)
+from lib.platforms import PlatformCTX, match_platform
+from lib.util import plot_scoreboard
+
+
+async def get_ctf_info(
+    interaction: discord.Interaction, name: Optional[str] = None
+) -> Optional[dict]:
+    # Trying to find ctf by channel category
```

Comment:
```suggestion
    # Attempt to find the CTF by its channel category.
```
Block and inline comments should be statements instead ([see PEP 8 standards](https://peps.python.org/pep-0008/#block-comments)).